packages/core/custom-elements-lsp/src/parser/analyze.ts

```diff
@@ -10,7 +10,7 @@ import {
   mixinParserConfigDefaults,
 } from '../main/custom-elements/manifest/repository.js';
 import { IOService } from '../main/utils/index.js';
-import { PluginConfig } from './analyze.types.js';
+import { PluginConfig } from '../main/plugins/plugins.types';
```

Comment:
eslint: `../main/plugins/plugins.types` import should occur before import of `../main/utils/index.js`